### PR TITLE
[single] Minor updates for modularity

### PR DESCRIPTION
**Pull request by kparichay**
Added minor updates in C-API single shot implementation for improved modularity
This makes the code skeleton at par with existing single-shot implementation

**Self evaluation:**
1. Build test: [ ]Passed [ ]Failed [*]Skipped
2. Run test: [ ]Passed [ ]Failed [* ]Skipped

Signed-off-by: Parichay Kapoor <pk.kapoor@samsung.com>

### CODE REVIEW

File: api/capi/src/nnstreamer-capi-single-new.c [hunk 1]
```diff
@@ -198,17 +198,86 @@ invoke_thread (void *arg)
...
+  ml_tensors_info_s *input_info;
+  GstTensorsInfo gst_info;
+  gchar *val;
+  guint rank;
+  gchar *prop_prefix, *prop_name;
+
+  if (is_input)
+    prop_prefix = g_strdup ("input");
+  else
+    prop_prefix = g_strdup ("output");
```
Comment (myungjoo) [inline api/capi/src/nnstreamer-capi-single-new.c]: These better be a static constant.
E.g., 
```
static const inputstr[] = "input";
static const outputstr[] = "ouput";
...
  prop_prefix = inputstr;
```

File: api/capi/src/nnstreamer-capi-single-new.c [hunk 2]
```diff
@@ -198,17 +198,86 @@ invoke_thread (void *arg)
...
+  /* allocate handle for tensors info */
+  ml_tensors_info_create (info);
+  input_info = (ml_tensors_info_s *) (*info);
+
+  ml_tensors_info_initialize (input_info);
+  gst_tensors_info_init (&gst_info);
+
+  /* get dimensions */
+  prop_name = g_strdup_printf ("%s", prop_prefix);
+  g_object_get (filter, prop_name, &val, NULL);
```
Comment (myungjoo) [inline api/capi/src/nnstreamer-capi-single-new.c]: property_name of g_object_get is const char *; thus, you don't need strdup. You can even directly give "input" here (or static const gchar *)

File: api/capi/src/nnstreamer-capi-single-new.c (hunk 1 above)
Comment (kparichay) [reply api/capi/src/nnstreamer-capi-single-new.c]: Updated with static string.

File: api/capi/src/nnstreamer-capi-single-new.c (hunk 2 above)
Comment (kparichay) [reply api/capi/src/nnstreamer-capi-single-new.c]: updated.

**Comment by kparichay** [comment]
> There is not too much dynamicity in string values. Let's use static-const gchar* rather than malloc-ed buffers. With a few tweaks, you can handle postfixes as well.

Updated with static definitions with #define and concat these strings defined with macro, which can be combined at compile time.

**Pull request merged**